src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -312,7 +314,69 @@ void ObjectMonitor::ClearSuccOnSuspend::operator()(JavaThread* current) {
 // -----------------------------------------------------------------------------
 // Enter support
 
+bool ObjectMonitor::enter_for(JavaThread* locking_thread) {
+  // Used by ObjectSynchronizer::enter_for to enter for another thread
```

Comment:
style bit: please end this sentence with a `;` and the lower case start of the next sentence will make sense.